spec/features/user_sees_cuisines_lists.spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "user sees a list of cuisines" do
+  scenario "see all cuisines" do
+    cuisine1 = Cuisine.create(name: "Chinese")
+    cuisine2 = Cuisine.create(name: "American")
+    cuisine3 = Cuisine.create(name: "Italian")
+
```

Comment:
Please begin using let statements for your model creation within tests 
https://relishapp.com/rspec/rspec-core/v/3-4/docs/helper-methods/let-and-let